src/main/grammars/RustParser.bnf

```diff
@@ -1446,6 +1446,24 @@ AssertMacroArgument ::= <<any_braces (AnyExpr [ ',' <<comma_separated_list Forma
 //noinspection BnfUnusedRule
 VecMacroArgument ::= <<any_braces ArrayInitializer >>
 //noinspection BnfUnusedRule
+AsmMacroArgument ::= <<any_braces [ <<comma_separated_list AsmMacroArg>> ] >>
+//noinspection BnfUnusedRule
+AsmMacroArg ::= AsmMacroOperand | STRING_LITERAL | AsmMacroOptions
+//noinspection BnfUnusedRule
+AsmMacroOperand ::= [ identifier '=' ] ( CONST AnyExpr | 'sym' PathWithoutTypeArgs | AsmMacroRegOperand )
+//noinspection BnfUnusedRule
+AsmMacroRegOperand ::= AsmMacroRegDir '(' AsmMacroRegSpec ')' AsmMacroRegTarget [ FAT_ARROW AsmMacroRegTarget ]
+//noinspection BnfUnusedRule
+AsmMacroRegDir ::= MetaVarIdentifier
```

Comment:
Let's not use `MetaVarIdentifier` here. It brings unreasonable PSI tree nesting. You can use `<<macroIdentifier>>` directly

Reply:
I tried it, but the rule then had no attributes. But I suppose it's not needed, I will just read `text` directly anyway.

Reply:
> I tried it, but the rule then had no attributes

They can be added with extension methods if needed